bin/zsnap

```diff
@@ -39,15 +40,27 @@ else:
 set_verbose(opts.verbose)
 
 snapshot_prefix = opts.prefix
-snapshot_postfix = lambda: time.strftime(opts.timeformat)
+snapshot_postfix = lambda: time.strftime(opts.timeformat,time.gmtime())
+
+snapshot_properties = {}
+
+if opts.property:
+	split = opts.property.split('=')
+	if len(split) != 2:
+		parser.error('--property should be of the form name=value')
+		sys.exit(os.EX_USAGE)
+	snapshot_properties[ split[ 0 ] ] = split[ 1 ]
 
 #===================== end configuration =================
 
 # ================ start program algorithm ===================
 
-src_conn = ZFSConnection(source_host)
+src_conn = ZFSConnection(source_host,properties=snapshot_properties.keys())
 snapshot_unique_name = snapshot_prefix + snapshot_postfix()
-flt = lambda x: x.name.startswith(snapshot_prefix)
+name_flt = lambda x: x.name.startswith(snapshot_prefix)
```

Comment:
these four lines can be simplified to:

flt = lambda x: x.name.startswith(snapshot_prefix) and (not snapshot_properties or x.get_property(snapshot_properties.keys()[0])==snapshot_properties.values()[0])